src/data/GeneratorList.js

```diff
@@ -1,68 +1,76 @@
-        formLabel: 'Paragraph:',
-        formName: 'longParagraph',
-        placeholder: 'paste your text here'
-    }
-];
\ No newline at end of file
+  {
+    id: 1,
+    title: "Blog Post Ideas",
+    description: "Stuck in the idea phase? Generate ideas",
+    link: "blog-ideas",
+    prompt: "Brainstorm an unordered list of blog post ideas for ",
+    description2:
+      "Enter your your content topic to generate a list of blog post ideas.",
+    formLabel: "Topic:",
+    formName: "topic",
+    placeholder: "e.g. Finance",
+  },
+  {
+    id: 2,
+    title: "Product Description",
+    description: "Generate an awesome Product Description",
+    link: "product-description",
+    prompt: "Write a persuasive and exciting product description for: ",
+    description2:
+      "Enter your your product and a few keywords to include in your output. Click Submit and get a product description generated for you by AI. Think less and sell more.",
+    formLabel: "Product Name & Purpose:",
+    formName: "productName",
+    placeholder: "e.g. ScoobySnacks will make your dog chill out",
+  },
+  {
+    id: 3,
+    title: "Company Bio",
+    description: "Generate an awesome Company Bio",
+    link: "company-bio",
+    prompt: "Write a persuasive and exciting company bio for: ",
+    description2:
+      "Enter your company name and a few keywords that you'd like to include in your output. Click Submit and get a company bio generated for you by AI. Think less and sell more.",
+    formLabel: "Company Name & Purpose:",
+    formName: "companyName",
+    placeholder: "e.g. Dunder Mifflin the best paper company ",
+  },
+  {
+    id: 4,
+    title: "SEO Blog Intro",
+    description: "Generate an awesome SEO blog into paragraph",
+    link: "seo-blog-intro",
+    prompt:
+      "Write an uplifting and positive Blog intro paragraph with SEO keywords for the blog title: ",
+    description2:
+      "Enter your blog posts title and SEO keywords that you'd like to include in your output. Click Submit and get a blog post introduction paragraph generated for you by AI. Think less and publish more.",
+    formLabel: "Blog post title:",
+    formName: "blogTitle",
+    placeholder: "e.g. How to Make a Commit with Git",
+  },
+  {
+    id: 5,
+    title: "LinkedIn Job Description",
+    description: "Generate a knowledgeable LinkedIn Job Description",
+    link: "linkedin-job-description",
+    prompt:
+      "Write an informative and knowledgeable LinkedIn job description in the first person past tense for the job title: ",
+    description2:
+      "Enter your your job title, click submit, and get a job description generated for you by AI.",
+    formLabel: "Job Title:",
+    formName: "jobTitle",
+    placeholder: "e.g. Software Engineer specializing in AI",
+  },
+  {
+    id: 6,
+    title: "TL;DR",
+    description: "Summarize verbose text into a TL;DR.",
+    link: "/tldr",
```

Comment:
Thanks again for working on this! 

For the sake of consistency throughout the data, let's remove the `/` (forward slash) as the other generators don't have one.